sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -69,6 +73,7 @@ class SessionCatalog(
   /**
    * Format table name, taking into account case sensitivity.
    */
+  // TODO: Should we use it for temp function name?
```

Comment:
Think about it again. I feel it is not really useful to set case sensitive by default for function names. I will remove this TODO. Function names are case sensitive (i.e. the current behavior. ).